test/filecomp.c

```diff
@@ -54,6 +54,7 @@ main(void)
     assert(!test_file(ctx, "keymaps/bad.xkb"));
     assert(!test_file(ctx, "keymaps/syntax-error.xkb"));
     assert(!test_file(ctx, "keymaps/syntax-error2.xkb"));
+    assert(!test_file(ctx, "keymaps/empty-symbol-decl.xkb"));
```

Comment:
```suggestion
    assert(test_file(ctx, "keymaps/empty-symbol-decl.xkb"));
```

The test should pass! Please move this above.

Reply:
Fixed!